sphinx/environment/__init__.py

```diff
@@ -451,44 +515,6 @@ def check_dependents(self, app, already):
             if docname not in already:
                 yield docname
 
-    def update_config(self, config, srcdir, doctreedir):
```

Comment:
Note: I didn't add DeprecationWarning here. Because both `update_config` and `_update_settings` are not released yet. They are only in master branch.

Reply:
OK